src/components/RoomView/Clue.tsx

```diff
@@ -65,16 +69,18 @@ const Clue = () => {
         playerSubmitting: playerId,
       };
       const newClueKey = push(child(ref(database), 'clues')).key;
-
-      const updates = {};
-      updates[newClueKey] = clueData;
-
-      dispatch(setCurrentClue(clueData));
-      update(gameHistoryRef, updates);
+      if (newClueKey) {
```

Comment:
Why did you have to put this in an if?

Reply:
because otherwise, typescript will give me an error since newClueKey could be undefined and we are using  updates[newClueKey] = clueData on the next line

Reply:
oh! Got it!